LICENSE

```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Sanctuary
```

Comment:
Let's add Copyright for Simon (unless he objects), I didn't reference the source until the final stages but the API design is his.